templates/repairs_mechanic.html

```diff
@@ -1 +1,63 @@
-{% include 'layout.html' %}
\ No newline at end of file
+{% include 'layout.html' %}
+
+<div class="corner-title">Umówione naprawy</div>
+
+<!-- add a new part -->
+<div style="background: #f39f32; margin-bottom: 3em; margin-top: 1em;">
+    <div style="text-align: center;"><strong>Dodaj nowy termin</strong></div>
+    <form action="" method="post" style="display: inline-block; margin-left: 35%;">
+        <select name="customer_id" id="customer_id">
+            <option value="" disabled selected>Wybierz klienta</option>
+            {% for customer in customers %}
+            <option id="customer_id" value="{{ customer.id }}">{{ customer.first_name }} {{ customer.last_name }}
+                {% endfor %}
+            </option>
+        </select>
+        <textarea id="car_name" name="car_name" rows="1" cols="30" placeholder="Nazwa Samochodu"></textarea>
+        <label for="start_of_repair">Start Naprawy:</label>
+        <input type="date" id="start_of_repair" name="start_of_repair">
+        <label for="end_of_repair">Koniec Naprawy:</label>
+        <input type="date" id="end_of_repair" name="end_of_repair">
+        <input style="display: inline-block; padding-top: 1.5em;" type="submit" value="Dodaj termin">
+    </form>
+</div>
+
+<div class="wrapper" style="background: gainsboro; padding-left: 7em;">
+    <div style="display: inline-block; width: 29.5%; min-width: 200px;">Nazwa samochodu</div>
+    <div style="display: inline-block; width: 10%; min-width: 100px;">Data startu</div>
+    <div style="display: inline-block; width: 10%; min-width: 100px;">Data końca</div>
+    <div style="display: inline-block; width: 10%; min-width: 100px;">Kwota</div>
+</div>
+
+{% if repairs %}
+<div class="scrollable-mechanic-list">
```

Comment:
![Zrzut ekranu 2024-03-18 181538](https://github.com/PatrykDu/popcorn-works-fastapi-fullstack-app/assets/113666877/b2d9c26d-7e26-46c0-b0e2-6dc01428fd09)

In my opinion there shouldn't be that much space between new row of the table. Maybe every even row can have diffrent color (gray or white)